internal/backend/sessions/sessionworkflows/workflows.go

```diff
@@ -225,20 +225,14 @@ func (ws *workflows) stopped(sessionID sdktypes.SessionID) {
 		}
 	}
 
-	if err := ws.svcs.DB.UpdateSessionState(ctx, sessionID, sdktypes.NewSessionStateStopped(reason)); err != nil {
-		z.Error("update session", zap.Error(err))
-	}
+	_ = ws.updateSessionState(ctx, sessionID, sdktypes.NewSessionStateStopped(reason))
```

Comment:
Why did you remove the error check here?

Reply:
I don't think I did. 
1. There was duplicated code and already existing method which updatesSessionState and logs
2. On error the only thing happened is log, and it's done now in single method - updateSessionState